Packs/CommonTypes/IndicatorFields/indicatorfield-Product.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id": "indicator_product",
```

Comment:
Can we use the existing incident_vendorproduct instead? and changing to "all types" as you did for the 'category' field.

Reply:
no, its an indicator field and `incident_vendorproduct` is an incident field.

Reply:
![image](https://github.com/user-attachments/assets/1c5c13e2-b5f6-4a48-8384-e7620772167b)
That what confused me, should the incidentfield-category be in that folder?

Reply:
Old indicator fields files start with "incident" because when created it was duplicated from incidents. It is also not a new field, just changed it to accept all indicator types.